venia-integration-tests/cypress.config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "baseUrl": "https://develop.pwa-venia.com/",
+    "defaultCommandTimeout": 10000,
+    "env": {},
+    "fixturesFolder": "src/fixtures",
+    "integrationFolder": "src/tests",
+    "pageLoadTimeout": 30000,
+    "requestTimeout": 30000,
+    "testFiles": "**/*.spec.js",
+    "viewportHeight": 900,
+    "viewportWidth": 1440
```

Comment:
Can we have different configurations if we wanted to run mobile- or desktop-specific tests?

Reply:
The problem is we don't have a single definition of mobile so I left it here. Depending on the tests we can set the viewport dimensions per test instead of a global value.